src/actions/articleActions.js

```diff
@@ -34,3 +32,47 @@ export const postArticle = postData => dispatch => {
       toast.error(errorMessage, { autoClose: false, hideProgressBar: true });
     });
 };
+
+export const fetchArticles = () => dispatch => {
+  axiosInstance
+    .get('/api/article/')
+    .then((response) => {
+      dispatch(getAllArticles(response.data.article));
+    });
+};
+
+export const fetchSpecificArticle = slug => dispatch => {
+  axiosInstance
+    .get(`/api/articles/${slug}`)
+    .then((response) => {
+      dispatch(getSpecificArticle(response.data));
+    });
+};
+
+export const fetchUserArticles = () => dispatch => {
+  axiosInstance
+    .get('/api/article/my-articles/')
+    .then(response => {
+      dispatch(getUserArticles(response.data.article));
+    });
+};
+export const updateArticle = (slug, newData) => dispatch => {
+  toast.dismiss();
+  return axiosInstance
+    .put(`/api/articles/${slug}/`, newData)
+    .then(response => {
+      dispatch(editArticleSuccess(true));
+      toast.success(
+        response.data.message,
+        { autoClose: 3500, hideProgressBar: true },
+        {
+          position: toast.POSITION.TOP_CENTER,
```

Comment:
@samuelbwambale , please remove this line, the toast position was already set.

Reply:
@MuhanguziDavid Line removed to take the default toast settings